index.md

```diff
@@ -8,6 +8,469 @@ This is some welcome text.
 
 This is some helpful description of the C0111 error.
 
+### C0102 Blacklisted name {#C0102}
+
+This error occurs when a variable name is listed in the black list. The black 
+list includes names like foo, bar, baz, toto, tata, tutu and so on. Students 
+are expected to use good variable names such as i, j, k, ex, Run. The example 
```

Comment:
Your example variable names are not particularly good. Don't give any examples, just mention that students should use *meaningful* variable names.